recipes/tinyxml/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "tinyxml" %}
+{% set version = "2.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://downloads.sourceforge.net/project/tinyxml/tinyxml/2.6.2/tinyxml_2_6_2.tar.gz
+    sha256: 15bdfdcec58a7da30adc87ac2b078e4417dbe5392f3afb719f9ba6d062645593
+    patches:
+      # These two patches are taken from the debian packaging of tinyxml.
+      #   The first patch enforces use of stl strings, rather than a custom string type.
+      #   The second patch is a fix for incorrect encoding of elements with special characters
+      #   originally posted at https://sourceforge.net/p/tinyxml/patches/51/
+      - enforce-use-stl.patch
+      - entity-encoding.patch
+  # This adds a CMakeLists.txt file to build a shared library and provide an install target
+  # submitted upstream as https://sourceforge.net/p/tinyxml/patches/66/
+  - path: CMakeLists.txt
+  
+
+build:
+  number: 0
+  skip: true  # [win]
+
```

Comment:
should tinyxml have a run_exports section? Since it's a shared library I believe yes.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements

Reply:
Yes, it should.

Reply:
Ok, I'll add it right away.